rust/hyperlane-base/src/settings/parser/json_value_parser.rs

```diff
@@ -26,7 +27,7 @@ impl<'v> ValueParser<'v> {
 
     /// Get a value at the given key and verify that it is present.
     pub fn get_key(&self, key: &str) -> ConfigResult<ValueParser<'v>> {
-        self.get_opt_key(key)?
+        self.get_opt_key(&key.to_case(Case::Flat))?
```

Comment:
Should we store `Case::Flat` in a global const?

Reply:
I don't think that makes sense in this specific case because it would not support any other case.